src/cudamatrix/cu-kernels-ansi.h

```diff
@@ -30,6 +30,9 @@
 #include "cudamatrix/cu-matrixdim.h"
 
 #if HAVE_CUDA == 1
```

Comment:
@pegahgh, can  you please review in detail?

On Wed, Apr 13, 2016 at 2:34 PM, Ke Li <notifications@github.com> wrote:

> In src/cudamatrix/cu-kernels-ansi.h
> <https://github.com/kaldi-asr/kaldi/pull/663#discussion_r59630010>:
>
> > @@ -30,6 +30,9 @@
> >  #include "cudamatrix/cu-matrixdim.h"
> >
> >  #if HAVE_CUDA == 1
>
> just finished modifying the code based on your comments.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/663/files/753c6be88875404a0f9a2eb609887dcf67724384..b5be620c91de8ec96b55d993d005e5d307b5d199#r59630010>
>